lib/driver.js

```diff
@@ -11,7 +11,8 @@ const CD_CONSTRAINTS = /** @type {const} */ ({
   logPath: { isString: true },
   autodownloadEnabled: { isBoolean: true },
   disableBuildCheck: { isBoolean: true },
-  browserName: { isString: true }
+  browserName: { isString: true },
+  webSocketUrl: { isBoolean: true },
```

Comment:
should we add this property to the base driver as it is defined by the standard?

Reply:
it's already there!